### PR TITLE
Remove "logger" field from config.json

### LINKED ISSUE #5176: Remove "logger" field from config.json
Remove "logger" field from config.json. Support for logging to a file needs to be removed as part of the improvements we are making to server side logging. 

The changes that need to be made are
1. Change the config.json structure version from 20 to 21.
2. Upgrade the structure from 20 to 21 upon startup.
3. Make sure that the unit tests reflect this change.
4. Add structure version 21.
5. Remove file-logger.go

### PR DESCRIPTION
**Pull request by kannappanr**
File logging is removed as part of improvement to server logging.

Change the config.json structure version from 20 to 21.

Upgrade the config.json structure to version 21 upon startup.

Added unit tests to cover this change.

Fixes #5176

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: cmd/common-main.go [hunk 1]
```diff
@@ -37,19 +37,9 @@ func checkUpdate(mode string) {
...
-		fatalIf(err, "Unable to initialize file logger")
-		log.AddTarget(fileLogTarget)
-	}
-
-	consoleLogTarget := globalServerConfig.Logger.GetConsole()
-	if consoleLogTarget.Enable {
-		InitConsoleLogger(&consoleLogTarget)
-	}
-
+func enableConsoleLogger() {
```
Comment (balamurugana) [inline cmd/common-main.go]: 1. If console is only logger, nothing to enable i.e. its already enabled in `logger.go`
2. Aren't you removing `logrus` usage?

Comment (kannappanr) [reply cmd/common-main.go]: I am not removing logrus usage in this PR. I have incorporated your first comment.

**Review by krishnasrinivas — DISMISSED** [review]
lgtm, tested

**Review by ebozduman — DISMISSED** [review]

**Review by balamurugana — COMMENTED** [review]
remove `type loggers` in logger.go which is stale without config entry.

**Review by ebozduman — APPROVED** [review]

**Review by krishnasrinivas — APPROVED** [review]

**Pull request merged**